P5/Source/Specs/conversion.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="conversion">
+  <gloss versionDate="2019-06-25" xml:lang="en">describes a mathematical conversion from one system of measurement to another.</gloss>
```

Comment:
The description is provided within the `<gloss>` element instead of `<desc>`;

Reply:
Done! :-)